src/Templates/src/templates/maui-mobile/PageModels/ManageMetaPageModel.cs

```diff
@@ -0,0 +1,101 @@
+using System.Collections.ObjectModel;
+using CommunityToolkit.Mvvm.ComponentModel;
+using CommunityToolkit.Mvvm.Input;
+using MauiApp._1.Data;
+using MauiApp._1.Models;
+using MauiApp._1.Services;
+
+namespace MauiApp._1.PageModels;
+
+public partial class ManageMetaPageModel : ObservableObject
+{
+	private readonly CategoryRepository _categoryRepository;
+	private readonly TagRepository _tagRepository;
+
+	[ObservableProperty]
+	private ObservableCollection<Category> _categories = [];
+
+	[ObservableProperty]
+	private ObservableCollection<Tag> _tags = [];
```

Comment:
```suggestion
	public ObservableCollection<Tag> Tags = [];
```

Reply:
https://github.com/dotnet/maui/pull/25419#discussion_r1813165803

Reply:
Leaving this in place for now